### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by devoncarew**
@alexander-doroshko

### CODE REVIEW

File: Dart/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Prerequisites:
 
 4. Open intellij-community project, compile it.
    Open File | Project Structure | Modules | `[+]` | Import Module, select intellij-plugins/Dart/Dart-community.iml.
-   In the same Project Structure dialog open the Dependencies tab of the `intellij.idea.community.main` module,
+   In the same Project Structure dialog open the Dependencies tab of the `intellij` > `idea.community` > `main` module,
```
Comment (alexander-doroshko) [inline Dart/README.md]: Strictly speaking, the name of the module is `intellij.idea.community.main`. Tree presentation can even be disabled in the context menu (Flatten Modules). How about 
> open the Dependencies tab of the `intellij.idea.community.main` module (`intellij` > `idea.community` > `main` node)

or something else, both correct and clear?

Comment (devoncarew) [reply Dart/README.md]: Makes sense. I needed to do some digging in the tree structure to find the right module, so calling out both the name and the location would be useful

**Comment by devoncarew** [comment]
updated -

**Comment by alexander-doroshko** [comment]
Pushed, thanks!
BTW are there any updates to the VM Service files that we should take?

**Comment by devoncarew** [comment]
Yes! They're on their way :)

**Comment by alexander-doroshko** [comment]
https://github.com/JetBrains/intellij-plugins/commit/3d5c204cc1d2582a12305a96232f2c4296a391bc

**Pull request closed without merging**